src/OrchardCore.Modules/OrchardCore.Microsoft.Authentication/Configuration/AzureADSettingsConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.Extensions.Options;
+using OrchardCore.Microsoft.Authentication.Services;
+using OrchardCore.Microsoft.Authentication.Settings;
+
+namespace OrchardCore.Microsoft.Authentication.Configuration;
+
+public class AzureADSettingsConfiguration : IConfigureOptions<AzureADSettings>
```

Comment:
Try to configure an app (dummy parameters are suitable) with this from appsettings:

```
    "OrchardCore_Microsoft_Authentication_AzureAD": {
      "DisplayName": "Dummy AD login",
      "AppId": "dummy",
      "TenantId": "dummy",
      "CallbackPath": "/signin-oidc"
    }
```

This won't show the external login button the login screen, unless you also configure one from the admin.

Test all the login providers in a similar way.

Reply:
> This won't show the external login button the login screen, unless you also configure one from the admin.

Let me check ..

Reply:
@Piedone regarding this one if you called `ConfigureAzureADSettings()` the login button will show up now, but if you are using the admin settings it will show up once you configure the options

Reply:
The point is, that if you configure nothing from the admin, then using `ConfigureAzureADSettings()` alone should make external login possible. If you don't use it, then the admin configuration alone. If you use both, then `ConfigureAzureADSettings()` should override the admin configuration. Is this what's happening?

Reply:
All the cases have been tested on Azure AD, I might need to do it with other social AUTH provider, but it should be the same after the latest changes